tests/test-cases/Group1-Docker-Commands/1-10-Docker-PS.robot

```diff
@@ -108,6 +108,9 @@ Docker ps powerOn container OOB
 
     Wait Until Keyword Succeeds  10x  6s  Assert Number Of Containers  ${len+1}
 
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop jojo
```

Comment:
I'm not sure why this is needed still... how does having a shared VCH require this? If there's a bug here, does it not exist in our current integration test suite?

Reply:
There is a bug that hasn't been caught before.  To get around this bug, I must stop the container here.  The PS test doesn't clean up their tests, and I have a clean up routine that tries to stop and delete all containers at the end of each suite, but it fails because we stopped the container out of band here.